src/win/fs.c

```diff
@@ -1989,6 +1992,9 @@ int uv_fs_open(uv_loop_t* loop, uv_fs_t* req, const char* path, int flags,
 
 
 int uv_fs_close(uv_loop_t* loop, uv_fs_t* req, uv_file fd, uv_fs_cb cb) {
+  if (req == NULL)
```

Comment:
I see every function looks like this now on Windows:

```
if (req == NULL)
  return UV_EINVAL;

 uv_fs_req_init(loop, req, UV_FS_FOO, cb);
```

Maybe use a macro to save some lines?